examples/clusters/model.jl

```diff
@@ -110,9 +112,7 @@ function generate_planet_periods_sizes_masses_in_cluster( star::StarAbstract, si
    log_mean_P = 0.0 # log(generate_periods_power_law(star,sim_param))
    # Note: Currently, drawing all periods within a cluster at once and either keeping or rejecting the whole cluster
    #       Should we instead draw periods one at a time?
-   const min_period::Float64 = get_real(sim_param,"min_period")
-   const max_period::Float64 = get_real(sim_param,"max_period")
-   Pdist = Truncated(LogNormal(log_mean_P,sigma_logperiod_per_pl_in_cluster*n),min_period,max_period)
+   Pdist = LogNormal(log_mean_P,sigma_logperiod_per_pl_in_cluster*n)
```

Comment:
When I ran the model with those lines, it produced too few planets and too many long period planets. Then I realized that it doesn't make sense to truncate Pdist between min and max period because at that point, the periods have not been scaled by a period_scale yet.

The cuts in min and max period are applied after multiplying by period_scale.